packages/payload/src/fields/baseFields/timezone/timezoneOptions.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * List of supported timezones
+ *
+ * label: UTC offset and location
+ * value: IANA timezone name
+ *
+ * @example
+ * { label: '(UTC-12:00) International Date Line West', value: 'Etc/GMT+12' }
+ */
+export const timezoneOptions = [
```

Comment:
These should probably be configurable at the top-level, such as under a new `admin.timezone` property.